src/main/java/uk/gov/companieshouse/overseasentitiesapi/converter/Version1_0Transformer.java

```diff
@@ -0,0 +1,32 @@
+package uk.gov.companieshouse.overseasentitiesapi.converter;
+
+import org.bson.Document;
+import org.springframework.stereotype.Component;
+import uk.gov.companieshouse.overseasentitiesapi.model.SchemaVersion;
+import uk.gov.companieshouse.overseasentitiesapi.utils.ApiLogger;
+
```

Comment:
i'd add some class level comments to explain what is changed/moved during a version 1.0 transform

Reply:
Yep. Good suggestion.